PackageBuilder/Build.cs

```diff
@@ -275,6 +282,80 @@ ListingSource MakeListingSourceFromManifest(VRCPackageManifest manifest)
                 Serilog.Log.Information($"Saved Listing to {savePath}.");
             });
 
+        async Task LoadPackageInfo(Dictionary<string, VpmPackageInfo> listSourceVpmPackages, List<VRCPackageManifest> packages)
+        {
+            var packagesByRepository = new Dictionary<string, List<(string id, VpmPackageInfo info)>>();
+
+            // collect packages by repository to reduce request per repository
+            foreach (var (packageId, package) in listSourceVpmPackages)
+            {
+                if (package.sources == null || package.sources.Length == 0)
+                {
+                    Serilog.Log.Warning($"Source repositories for {packageId} is not defined! This package will be ignored.");
+                    continue;
+                }
+
+                foreach (var sourceUrl in package.sources)
```

Comment:
I don't have actual use case but it's easy enough so I make it list of repositories.

Reply:
In that case we can always make a change when the use case apears.
I think we my as well keep the interface simple to avoid confusing people with it.
Else what could you write in the documentation about it.

Reply:
I think it's simple enough but I'll make it string.